tests/trx_generator/trx_provider.hpp

```diff
@@ -2,9 +2,13 @@
 
 #include<eosio/chain/transaction.hpp>
 #include<eosio/chain/block.hpp>
-#include<boost/asio/ip/tcp.hpp>
-#include<fc/network/message_buffer.hpp>
 #include<eosio/chain/thread_utils.hpp>
+
+#include<fc/network/message_buffer.hpp>
```

Comment:
Is `message_buffer.hpp` required in this header file?

Reply:
No.